.github/workflows/pull-request.yml

```
@@ -0,0 +1,666 @@
+          # Check if any unauthorized files changed
+          if [ ! -z "$CHANGED_UNAUTH_FILES" ]; then
+            echo "any_changed=true" >> $GITHUB_OUTPUT
+            echo "any_deleted=true" >> $GITHUB_OUTPUT
+          else
+            echo "any_changed=false" >> $GITHUB_OUTPUT
+            echo "any_deleted=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: List all changed unauthorized files
+        if: steps.changed-unauth-files.outputs.any_changed == 'true' || steps.changed-unauth-files.outputs.any_deleted == 'true'
+        env: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)